@IliaF 

`  E:\Jenkins\workspace\MITK-PR\Modules\Segmentation\Algorithms\mitkShowSegmentationAsElasticNetSurface.cpp(53): error C2171: '!': illegal on operands of type 'void' [E:\BPM\SB\MITK-superbuild\MITK-build\Modules\Segmentation\MitkSegmentation.vcxproj] [E:\BPM\SB\MITK-superbuild\MITK-build.vcxproj] [E:\BPM\SB\MITK.vcxproj]`

--- a/Modules/Segmentation/Algorithms/mitkShowSegmentationAsElasticNetSurface.h
+++ b/Modules/Segmentation/Algorithms/mitkShowSegmentationAsElasticNetSurface.h
@@ -61,7 +61,7 @@ private:
   ShowSegmentationAsElasticNetSurface(const Self&);
   void operator=(const Self&);
 
-  void calcRegion();
+  bool calcRegion();
   void createSurfaceCubes();
   void createNodes();
   void linkNodes();

